### ISSUE TITLE
Verify matrix webhook session

### ISSUE THREAD

**Issue by casper294** [immanuelfodor/matrix-encrypted-webhooks#5]
Is it possible to verify the matrix session from this webhook? As all messages sent from the webhook currently have a warning that the session isn't verified.

<img width="565" alt="image" src="https://github.com/user-attachments/assets/8afd5261-e0cd-4a64-8bdd-c320d8877578">

I tried verifying from another session in my browser, but that does send a message to this client, but I don't see a way to receive it?

**Comment by casper294** [comment]
Tried to manually enter the device_id and access_token in the credentials.json as described here: https://github.com/immanuelfodor/matrix-encrypted-webhooks/issues/1#issuecomment-900852819

The error message under the red exclamation mark changed from: `Encrypted by a device not verified by its owner` to `Encrypted by an unknown or deleted device. `

**Comment by immanuelfodor** [comment]
It's possible to do [manual](https://matrix-nio.readthedocs.io/en/latest/examples.html#manual-encryption-key-verification) or [emoji](https://matrix-nio.readthedocs.io/en/latest/examples.html#interactive-encryption-key-verification) verification with the [Nio](https://github.com/matrix-nio/matrix-nio) library that this project uses, but it's not (yet) implemented in the [E2EEClient.py](https://github.com/immanuelfodor/matrix-encrypted-webhooks/blob/main/src/E2EEClient.py), so you can't do it today. The linked comment is about another topic with SSO login, it's unrelated to yours.

Why I haven't implemented verification?
- I'm the only user
- The whole thing runs in a private network with no external access
- Only Element is reluctant to complain about trust, better clients, such as SchildiChat don't complain
- Even larger projects, such as Mautrix bridges don't implement it, so if it seems "good enough" to them, then it's good enough to me :grinning: 

But I'm happy to accept a PR if you (or anybody else) add verification to the project! :pray:

**Comment by casper294** [comment]
For some reason I had verification working without issues on a previous server installation. Just moved it to a new server this week and for some reason can't get it to appear as verified anymore. 

<img width="576" alt="image" src="https://github.com/user-attachments/assets/4d949e46-dcdc-4c96-b91b-d3004ab3c152">

Really appreciate your response. Just wanted to make sure there wasn't an easy fix I'm overlooking.

**Comment by immanuelfodor** [comment]
Yeah, nothing to worry about, the bot messages are still end-to-end encrypted, it's just a "cosmetic annoyance" by the Element clients.
